### PR TITLE
Re-Structure the SDK design

### PR DESCRIPTION
**Pull request by Arash-Swan**
Re-Structure the SDK design

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,61 @@
...
+- **Data Models**: Leverage our pre-built models for spaces, jobs, and users.
+- **Service Abstractions**: Gain access to high-level functionalities through our service layer.
+- **Comprehensive Documentation**: Discover detailed guides and references in the `docs/` directory.
+
+## Installation
+
+Install the SDK with ease:
+
+```bash
+pip install sdk-name
```
Comment (DanielJiangCloud) [inline README.md]: you may want to change this to swan-sdk

File: README.md [hunk 2]
```diff
@@ -1 +1,61 @@
...
+## Overview
+
+The SWAN SDK provides a streamlined and efficient interface for interacting with our API. It's tailored for easy management of spaces, jobs, and users, making it a versatile tool for a wide range of applications.
+
+## Features
+
+- **Easy API Integration**: Simplify your workflow with our user-friendly API client.
+- **Data Models**: Leverage our pre-built models for spaces, jobs, and users.
+- **Service Abstractions**: Gain access to high-level functionalities through our service layer.
+- **Comprehensive Documentation**: Discover detailed guides and references in the `docs/` directory.
```
Comment (DanielJiangCloud) [inline README.md:14]: I assume the docs/ directory will come later

File: test/tasks/test_celery_tasks.py [hunk 3]
```diff
@@ -2,7 +2,8 @@
 
 import pytest
 import requests_mock
-from swan.common import constants as c
+
+from scr.constants.constants import CELERY
```
Comment (DanielJiangCloud) [inline test/tasks/test_celery_tasks.py]: What is scr, should it be src?

**Review by DanielJiangCloud — CHANGES_REQUESTED** [review]
Looks good to push but please make the required changes before pushing

File: README.md (hunk 2 above)
Comment (Arash-Swan) [reply README.md:14]: I created the structure, we will add everything in steps

File: README.md (hunk 1 above)
Comment (Arash-Swan) [reply README.md]: done

File: test/tasks/test_celery_tasks.py (hunk 3 above)
Comment (Arash-Swan) [reply test/tasks/test_celery_tasks.py]: yes i made necessary changes

**Review by DanielJiangCloud — APPROVED** [review]
LGTM

**Pull request merged**